FirebaseCombineSwift/README.md

```diff
@@ -58,4 +58,54 @@ end
     .replaceError(with: "(unable to sign in anonymously)")
     .assign(to: \.uid, on: self)
     .store(in: &cancellables)
-```
\ No newline at end of file
+```
+#### Sign in with a given 3rd-party credentials 
+
+
+In the `sign(_:didSignInFor:withError:)` method, get a Google ID token and Google access token from the GIDAuthentication object and exchange them for a Firebase credential. Then, Asynchronously sign in to Firebase with the given tokens pair:
```

Comment:
I think the official docs say it quite nicely: 

> In the `signIn:didSignInForUser:withError:` method, get a Google ID token and Google access token from the `GIDAuthentication` object and exchange them for a Firebase credential:

Let's use this and change your paragraph to:

> In the `sign(_:didSignInFor:withError:)` method, get a Google ID token and Google access token from the `GIDAuthentication` object and asynchronously exchange them for a Firebase credential: